ZenPacks/zenoss/Import4/bin/install_pkg.sh

```diff
@@ -25,6 +25,14 @@ info_out "Installing import4 scripts from $progdir ..."
 status_out "initialize" "Installing import4 scripts ..."
 
 [[ -d "$VOL_D" ]]     || err_exit "The environment does not have the shared volume: $VOL_D" 
+
+# make all accessible to all
+/usr/bin/yum install acl -y >> "$imp4_tmp/install.log" 2>&1
```

Comment:
There are many directories + files under /import4 got created.. It is shared by root:root and zenoss:zenoss from two service images.  This is a cleaner and more solid way of handling these shared files.